projects/angle/fuzzer_profile

```diff
@@ -21,10 +22,14 @@ if (angle_has_build) {
 
     ldflags = [
       "-fsanitize=" + getenv("SANITIZER"),
-      "-L/usr/lib/x86_64-linux-gnu",
-      "-L/usr/lib/gcc/x86_64-linux-gnu/9",
+      "-fprofile-instr-generate"
     ]
 
+    if (res_dir != "") {
+      cflags += [ "-resource-dir=" + res_dir ]
+      ldflags += [ "-resource-dir=" + res_dir ]
+    }
+
```

Comment:
Are the changes for `angle` supposed to be here?

Reply:
thats an accident wrong commit.